### ISSUE TITLE
Dashboard should display "message" field from db rather than always "Could not capture snapshot"

### ISSUE THREAD

**Issue by jerclarke** [berkmancenter/amber_wordpress#46]
Found this while writing up #44  but it's its own issue IMHO

When a post fails the `Notes` column on the dashboard always seems to say "Could not capture snapshot", even if the `message` field in the `amber_check` table of the database has a much more useful message. 

What's up with that? Why wouldn't I want to see the extra info?

In my case, both Twitter (#44) and Facebook (#45) URLs are regularly broken because Amber is obeying the `robots.txt` files of those domains. Separate from that being annoying, is that I had to look in the database directly to find that Amber knew exactly why they were broken (`message`: "Blocked by robots.txt") but the Amber Dashboard didn't share that info with me. 

This is excruciatingly useful info for someone debugging a `Down` state in the Dashboard, and it's actually useful to the user, unlike  "Could not capture snapshot", which it turns out is the exact same information as the `Down/Up` info in the `Status` column. 

So to me it seems obvious: Dashboard should list the `message` value from the database if there is one, and only resort to the redundant  "Could not capture snapshot" if for some reason the URL is `Down` but `message` has no info. 

Thanks for your attention and help

**Comment by jlicht** [comment]
@jerclarke Thanks for this, and your other recent suggestions. We don't have a specific timeline for the next release, but will certainly take these into account.

**Comment by jerclarke** [comment]
@jlicht Thank you for your reply. I know I came late to Amber, after initial development energy has died down. I'm hoping we can work together on this now that Global Voices (also a Berkman baby) is finally in a good place to work on our implementation (we were in the midst of major server revisions when Amber first came out).
